apps/extension/src/get-rpc-impls.ts

```diff
@@ -24,36 +24,40 @@ import { viewImpl } from '@penumbra-zone/router/src/grpc/view-protocol-server';
 import { localExtStorage } from '@penumbra-zone/storage/src/chrome/local';
 import { ServiceType } from '@bufbuild/protobuf';
 
-const grpcEndpoint = await localExtStorage.get('grpcEndpoint');
+export const getRpcImpls = async () => {
```

Comment:
Wrapped `rpcImpls` in a `getRpcImpls` function so that we can wait to call it until we know the gRPC endpoint exists in local storage. (View this file in "hide whitespace" mode — there are almost no changes.)